pkg/collector/gadgets/dns_tracer_collector_test.go

```diff
@@ -0,0 +1,80 @@
+package gadgets_test
```

Comment:
Is this a golang convention, to put tests in a different package? So far, I've just been creating `<filename>_test.go` in the same package. Is the goal to avoid accessing package internals from tests?